app/models/application_draft.rb

```diff
@@ -75,6 +75,8 @@ class ApplicationDraft < ApplicationRecord
   belongs_to :updater, class_name: 'User'
   belongs_to :project1, class_name: 'Project'
   belongs_to :project2, class_name: 'Project'
+  has_many   :application_draft_work_weeks
```

Comment:
Hm, I fear I don't get the benefits of this approach 🙈 

It appears rather complex / overly complex to me for what we want to archive: have working-weeks set to either "full time" or "half time".

An more lightweight approach would for instance be: add a new integer column `work_weeks`to the `application_draft` model, make it an [Rails enum](http://api.rubyonrails.org/classes/ActiveRecord/Enum.html) (defined with hash syntax at best) and add the respective translations for it and use a simple select in the form.

Or even simpler: make it a string column and use something like [this gem](https://github.com/makandra/assignable_values) to restrict the values and get the naming right.

Reply:
Hmm, another project where I was collaborated insisted on having this approach over others. Now I'm not sure which is best..

The definite advantage of having WorkWeeks be tied to a table is that we're not tied to hard-coded values and can later on add more options, say a 10 hour week (highly improbable)

Reply:
Yes, makes sense, I get this point!

I think enum defined with a hash will also save you in this case - so I'd consider it a similar flexible but a bit more lightweight option... What do you think?

Reply:
On the other hand, we have the `heard_about_it` field, which is a string array. I could go with this instead